frontend/src/components/header/Header.js

```diff
@@ -1,56 +1,59 @@
-                                    to="/"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >HOME</Link>
-                            </Grid>
-                            <Grid item xs={3}>
-                                <Link
-                                    to="/events"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >EVENTS</Link>
-                            </Grid>
-                            <Grid item xs={3}>
-                                <Link
-                                    to="/education"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >EDUCATION</Link>
-                            </Grid>
-                            <Grid item xs={3}>
-                                <Link
-                                    to="/ecosystem"
-                                    rel="noopener noreferrer"
-                                    className="text"
-                                >ECOSYSTEM</Link>
-                            </Grid>
-                        </Grid>
-                    </Grid>
-                    <Grid item xs={3}/>
-                </Grid>
-            </div>
-            );
-    }
-}
\ No newline at end of file
+  render() {
+    const { white: lightTheme, selected } = this.props
+    return (
+      <div className={this.props.white ? 'main-header-white' : 'main-header'}>
+        <Grid container>
+          <Grid item xs={3} className="header-logo">
+            <img src={lightTheme ? logoLight : logo} className={'logo'} alt={'logo'} />
+          </Grid>
+          <Grid item xs={6} className="menu-container">
+            <Grid container className="menu">
+              <Grid className="menu-item">
+                <Link to="/" rel="noopener noreferrer" className="text">
+                  HOME
+                </Link>
+              </Grid>
+              <Grid className="menu-item">
+                <Link
+                  to="/events"
+                  rel="noopener noreferrer"
+                  className={`text ${selected === 'events' ? 'selected' : ''}`}
+                >
+                  EVENTS
+                </Link>
+              </Grid>
+              <Grid className="menu-item">
+                <Link
+                  to="/education"
+                  rel="noopener noreferrer"
+                  className={`text ${selected === 'education' ? 'selected' : ''}`}
+                >
+                  EDUCATION
+                </Link>
+              </Grid>
+              <Grid className="menu-item">
+                <Link
+                  to="/ecosystem"
+                  rel="noopener noreferrer"
+                  className={`text ${selected === 'ecosystem' ? 'selected' : ''}`}
```

Comment:
Two things related to the <Link /> component:
- It does not accept a `rel` prop, and this attribute only goes for external links/anchors
- You can use [<NavLink />](https://reacttraining.com/react-router/web/api/NavLink) component that's the same as <Link /> but with props for custom styling.

In example:
```javascript
<NavLink
  to="/ecosystem"
  className="text"
  activeClassName="selected" // This will be applied only if we're on "/ecosystem"
>
 ECOSYSTEM
</NavLink>
```